src/arrays.js

```diff
@@ -1,61 +1,167 @@
 const getNthElement = (index, array) => {
-  // your code here
+  while (index >= array.length) {
+    index -= array.length;
+  }
+  return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  // let newString = `${array[0]},`;
+  // for (let i = 1; i < array.length; i++) {
+  //   i !== array.length - 1
+  //     ? (newString += `${array[i]},`)
+  //     : (newString += `${array[i]}`);
+  // }
+  // return newString;
+  ///////////////////////////////////////////
+  // let newString = '';
+  // for (let i = 0; i < array.length; i++) {
+  //   i !== array.length - 1
+  //     ? (newString += `${array[i]},`)
+  //     : (newString += `${array[i]}`);
+  // }
+  // return newString;
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  // const array = [string[0]];
+  // for (let i = 1; i < string.length; i++) {
+  //   if (i % 2 === 0) array.push(string[i]);
+  // }
+  // return array;
+
+  /////////////////////////////////////////////
+  // const array = [];
+  // for (let i = 0; i < string.length; i++) {
+  //   if (i % 2 === 0) array.push(string[i]);
+  // }
+  // return array;
+
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  // const newArray = [];
+  // for (let i = 0; i < array.length; i++) {
+  //   newArray.push(array[i]);
+  // }
+  // newArray.push(element);
+  // return newArray;
+
+  ////////////////////////////////////////
+  // const newArray = [];
+  // array.forEach(item => newArray.push(item));
+  // newArray.push(element);
+  // return newArray;
+
+  const newArray = array.map(item => item);
+  newArray.push(element);
+  return newArray;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  // for (let i = 0; i < array.length; i++) {
+  //   if (i === index) array.splice(i, 1);
+  // }
+
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  // const newArray = [];
+  // numbers.forEach(num => {
+  //   newArray.push(String(num));
+  // });
+  // return newArray;
+
+  return numbers.map(num => String(num));
```

Comment:
you can also use the inbuilt number method `.string` instead of `String(num)`. i.e.
```
return numbers.map(num => num.toString());
```